tests/components/pvpc_hourly_pricing/conftest.py

```diff
@@ -15,7 +19,10 @@
 def check_valid_state(state, tariff: str, value=None, key_attr=None):
     """Ensure that sensor has a valid state and attributes."""
     assert state
-    assert state.attributes[ATTR_UNIT_OF_MEASUREMENT] == f"€/{ENERGY_KILO_WATT_HOUR}"
+    assert (
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6184085)